spec/integ/crypto/crypto.spec.ts

```diff
@@ -55,6 +54,7 @@ import {
     Room,
     RoomMember,
     RoomStateEvent,
+    CryptoEvent,
```

Comment:
this was sorted before. :-1:  to this change.

Reply:
sry, messed up one of the rebase